docs/reference/api/docker_remote_api_v1.23.md

```diff
@@ -184,6 +195,10 @@ Query Parameters:
   -   `status=`(`created`|`restarting`|`running`|`paused`|`exited`|`dead`)
   -   `label=key` or `label="key=value"` of a container label
   -   `isolation=`(`default`|`process`|`hyperv`)   (Windows daemon only)
+  -   `ancestor`=(`<image-name>[:<tag>]`,  `<image id>` or `<image@digest>`)
+  -   `before`=(container's id or name)
```

Comment:
minor thing, but for consistency, should these use `< >`, e.g. `<container id>` instead of "container's id"  since you're using `<image id>` above?